querybook/webapp/redux/comment/reducer.ts

```diff
@@ -0,0 +1,255 @@
+import { produce } from 'immer';
+
+import { CommentAction, ICommentState } from './types';
+
+const initialState: ICommentState = {
+    cellIdToComment: {},
+    tableIdToComment: {},
```

Comment:
An alternative setup of the comment state could be like what we have in the database, 
cellIdToCommentId: {}
tableIdToCommentId: {}
commentsById: {id -> comment}

as all the operations of a comment, no matter it's cell comment or table comment, they are the same, we may save some duplicate actions and make the reaction operations more easy.

Reply:
ah great. will update to this